eng/Versions.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <!--Package versions-->
   <PropertyGroup>
-    <MicrosoftDotnetSdkInternalPackageVersion>6.0.100-preview.5.21260.2</MicrosoftDotnetSdkInternalPackageVersion>
-    <MicrosoftAndroidSdkWindowsPackageVersion>30.0.100-ci.main.2</MicrosoftAndroidSdkWindowsPackageVersion>
+    <MicrosoftDotnetSdkInternalPackageVersion>6.0.100-preview.5.21266.3</MicrosoftDotnetSdkInternalPackageVersion>
+    <MicrosoftAndroidSdkWindowsPackageVersion>30.0.100-ci.6-0-1xx-preview5.13</MicrosoftAndroidSdkWindowsPackageVersion>
```

Comment:
This version here appears to be before I added the correct `.preview.5` version, and it doesn't have our latest build:

https://github.com/xamarin/xamarin-android/commits/release/6.0.1xx-preview5

We might need to wait until another xamarin-android version is pushed to maestro.